runtime/compiler/runtime/RelocationRecord.hpp

```diff
@@ -39,346 +39,14 @@ class TR_RelocationTarget;
 struct TR_RelocationRecordBinaryTemplate;
 typedef TR_ExternalRelocationTargetKind TR_RelocationRecordType;
 
-// These *BinaryTemplate structs describe the shape of the binary relocation records.
```

Comment:
I am *so* happy to finally see these structs move out of the header file......Bravo!

Reply:
Done in https://github.com/eclipse/openj9/pull/11235/commits/93ac7a2b30dc085bb8432e177ad191816df6cc94